ios/Classes/TiMapView.h

```diff
@@ -30,6 +30,9 @@
     NSMutableArray *polygonProxies;
     NSMutableArray *circleProxies;
     NSMutableArray *polylineProxies;
+    
+    //selected annotation
+    MKAnnotationView<TiMapAnnotation> * selectedAnnotation;
```

Comment:
When you assign it with `selectedAnnotation = ann;`, it can be deallocated, since it's not using ARC. That's why you need to retain and release it.